docs/pages/management/operations/upgrading.mdx

```diff
@@ -83,22 +88,15 @@ precaution. This allows for a safe rollback in case the migration itself fails.
 
 </Admonition>
 
-When upgrading multiple clusters:
-
-1. First, upgrade the root cluster, i.e. the one that other clusters trust.
-2. Upgrade the Trusted Clusters.
-
 </TabItem>
 <TabItem scope={["cloud"]} label="Teleport Cloud">
 
-The Teleport Auth Service and Proxy Service are upgraded automatically. When
-upgrading resource services, you may upgrade in any sequence or at the same
+The Teleport Auth Service and Proxy Service are upgraded automatically within the specified
+window. When upgrading resource services, you may upgrade in any sequence or at the same
 time.
 
-When upgrading multiple clusters:
-
-1. First, upgrade the root cluster, i.e. the one that other clusters trust.
-2. Upgrade the Trusted Clusters.
+As documented in the Component compatibility the resource services should be the same major version or
```

Comment:
We refer to "the Component compatibility" as if someone should know what page that is. If they do already know what page we're talking about, they're probably familiar enough with things that they don't need this reminder.

Maybe something like this instead?

> Resource services should not be upgraded to a new major version
> until the Auth Service and Proxy Service have been upgraded.
> For more information on component compatibility, see <link>

I'm also not sure how I feel about the term "resource services." This is the first I've heard it.

Reply:
"Resource services" was one possible alternative we talked about when deciding on a general term for instances of the Kubernetes, SSH, etc. Service. The Core Concepts page uses "instance" or "agent" (https://goteleport.com/docs/core-concepts/#agent).